internal/services/environment_settings/resource_environment_settings_test.go

```diff
@@ -40,6 +40,7 @@ func TestAccTestEnvironmentSettingsResource_Validate_Read(t *testing.T) {
 						  is_audit_enabled             = true
 						  is_user_access_audit_enabled = true
 						  is_read_audit_enabled        = true
+						  log_retention_period_in_days = null
```

Comment:
as in previous comment "null" is not a proper value that the user should use. Its not defined at all or something between 31 - 99999

Reply:
I will set 31. 
From here: https://learn.microsoft.com/en-us/power-platform/admin/manage-dataverse-auditing#startstop-auditing-for-an-environment-and-set-retention-policy

the "Set the retention policy for these logs", Default value is: Forever
the "Set a custom retention policy", could be: Maximum: 24,855 days. Visible if you select "Custom" in the above setting.

24,855 days = +/- 68 years.